operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -1,52 +1,84 @@
 # Create a Session Template to Boot Compute Nodes with Scalable Boot Projection Service (SBPS)
 
 When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Scalable Boot Projection Service (SBPS)](../../glossary.md#scalable-boot-projection-service-sbps)
-projects the root file system (`rootfs`) over the network to the compute nodes.
+projects the root file system (`rootfs`) over the network to the compute nodes using iSCSI.
 
-This page covers the appropriate contents for a BOS session template in order to use SBPS.
+This page covers the necessary configuration of a BOS session template in order to use SBPS.
 
 - [Boot set `rootfs_provider` parameter](#boot-set-rootfs_provider-parameter)
-- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameters)
-    - [`<transport>`](#transport)
-    - [`<schema version>`](#schema-version)
-    - [`<IQN Domain>`](#iqn-domain)
-    - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
-    - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
-    - [`<ramroot>`](#ramroot)
-    - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
-- [Boot set S3 parameters](#boot-set-s3-parameters)
+- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameter)
```

Comment:
"parameters" can be changed to "parameter" in the link too.

Reply:
Fixed.